index.md

```diff
@@ -52,3 +52,9 @@ To experience the Checker Framework from within your browser, see our
 
 Please address your questions and comments to
 [Werner Dietl](https://ece.uwaterloo.ca/~wdietl/contact.html).
+
+## Contributors
+
+- [Credits](https://eisop.github.io/cf/manual/manual.html#credits)
```

Comment:
```suggestion
- [Credits](https://eisop.github.io/cf/manual/#credits)
```

I do like the shorter links. Yes, they currently expand to have `manual.html` in them, but that's an implementation detail that we could change later on.